docs/modeling/code-generation-in-a-build-process.md

```diff
@@ -305,7 +305,13 @@ If you update an included file or another file read by the template, Visual Stud
 ## See also
 
 
-::: moniker range=">=vs-2019"
+::: moniker range=">=vs-2022"
+
+- There's good guidance in the T4 MSbuild template at `%ProgramFiles(x86)%\Microsoft Visual Studio\2022\BuildTools\MSBuild\Microsoft\VisualStudio\v17.0\TextTemplating\Microsoft.TextTemplating.targets`
```

Comment:
`BuildTools`? How about we just stick with `Enterprise` here?

Reply:
@jmartens -What should be done here?

Reply:
> @jmartens -What should be done here?

I thought I replied by email to this conversation but apparently it did not come through. 

According to the comment of @mgoertz-msft:
> `BuildTools`? How about we just stick with `Enterprise` here?

The path needs to be adjusted for Enterprise edition, since I don't have access to Enterprise edition I can not lookup the correct path. Someone else perhaps?